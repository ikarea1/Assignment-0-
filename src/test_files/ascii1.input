In this World we can test the different allowed ASCII characters like  , 	 , 
 ,  ,  , 
and even characters like ! 
