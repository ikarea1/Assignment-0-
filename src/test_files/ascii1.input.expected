test_files/ascii1.input: data
